.github/workflows/publish.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish
+
+on:
+  push:
+    branches: [master]
+    tags: ['*']
+
+jobs:
+  publish-scm-1:
+    if: github.ref == 'refs/heads/master'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: tarantool/rocks.tarantool.org/github-action@master
+        with:
+          auth: ${{ secrets.ROCKS_AUTH }}
+          files: smtp-scm-1.rockspec
+
+  publish-tag:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+    env:
+      CMAKE_LDOC_FIND_REQUIRED: 'YES'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: rosik/setup-tarantool@v1
```

Comment:
If it only needed for `tarantoolctl rocks new_version/install/pack`, I would remove it and use `sed` (it is not the only reason to used sed, see below).

Reply:
Yes, removed